bin/simulator.py

```diff
@@ -369,21 +381,52 @@ def simulate(self, args):
         tree = self.init_node(args, args.naive_tseq.nuc, 0, None, target_seqs, mean_kd=args.naive_kd)
         if args.selection:
             self.tdist_hists[0] = self.get_target_distance_hist(args, tree)  # i guess the first entry in the other two just stays None
-
+            current_indices = self.get_scatter_plots(args, tree, current_time)
+            self.scatter_index.append(current_indices[1])
+            self.aff_index.append(current_indices[0])
+            self.bcr_exp_index.append(current_indices[2])
+            self.antigen_cap_index.append(current_indices[3])
         if args.debug == 1:
             print('        end of      live     target dist (%s)          kd            lambda       termination' % args.metric_for_target_distance)
             print('      generation   leaves      min  mean           min    mean      max  mean        checks')
         if args.debug > 1:
             print('       gen   live leaves   (%s: terminated/no children)' % selection_utils.color('red', 'x'))
             print('             before/after   ileaf   lambda  n children     n mutations            Kd   (%s: updated lambdas)' % selection_utils.color('blue', 'x'))
         static_live_leaves, updated_live_leaves = None, None
+        if args.n_initial_seqs > 1:
```

Comment:
I think this is a great start on --n_initial_seqs, but I think we don't really want to [duplicate this much](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code here.